frontend/src/lib/introductions/AttentionRequiredBanner.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { LinkButton } from 'lib/components/LinkButton'
+import { systemStatusLogic } from 'scenes/instance/SystemStatus/systemStatusLogic'
+import { useActions, useValues } from 'kea'
+import { CloseOutlined } from '@ant-design/icons'
+import clsx from 'clsx'
+import { announcementLogic } from '~/layout/navigation/TopBar/announcementLogic'
+import { userLogic } from 'scenes/userLogic'
+import { organizationLogic } from 'scenes/organizationLogic'
+import { OrganizationMembershipLevel } from 'lib/constants'
+
+export function AttentionRequiredBanner(): JSX.Element | null {
+    const { user } = useValues(userLogic)
+    const { currentOrganization, currentOrganizationLoading } = useValues(organizationLogic)
+    const { overview, systemStatusLoading } = useValues(systemStatusLogic)
+    const { shownAnnouncementType, closable } = useValues(announcementLogic)
+    const { hideAnnouncement } = useActions(announcementLogic)
+
+    const areAsyncMigrationsUpToDate = overview.find((metric) => metric.key === 'async_migrations_ok')
+    const isUserAdmin = user?.is_staff || currentOrganization?.membership_level === OrganizationMembershipLevel.Admin
+    const loadingData = systemStatusLoading || currentOrganizationLoading
+
+    if (loadingData || areAsyncMigrationsUpToDate || !isUserAdmin) {
```

Comment:
Hmm I actually disagree here.

The idea with components like `NewFeatureBanner` (in which this was inspired) is that to update an announcement, you specify everything within the component. It's a bit manual, but it's the way we've been doing things. 

I don't want to get this logic in a logic file because it would always be changing. This is very specific to what the current content of the announcement is, which is async migrations. Tomorrow this might be some security patch that requires some manual steps. Adding this to the logic would add too much noise there imo, and keeping it in the component limits the number of places where things need to be changed for a new announcement.

Reply:
I disagree (business logic doesn't belong in tsx + there's a consistency argument with other components here) but willing to drop this if you feel strongly on it.

Reply:
I disagree (business logic doesn't belong in tsx + there's a consistency argument with other components here) but willing to drop this if you feel strongly on it.

However one issue here is that there are many announcements competing for the same "announcement slot". In this case there can be _no_ feature announcements until this component is removed since we can't check whether the announcement is really needed or not in the logic that decides what announcement to show.

Reply:
Alright I've simplified the logic so we can keep everything in kea's `announcementLogic`